config/config.go

```diff
@@ -346,6 +355,10 @@ func (cfg Config) Validate() error {
 		)
 	}
 
+	if cfg.General.ApplicationName != "" {
+		err = validate.ASCII("General.ApplicationName", cfg.General.ApplicationName, 0, 32)
```

Comment:
Use `validate.Many(err, validate.ASCII(...` so we don't ignore/erase previous errors